cli/cmd/agent_installctf.go

```diff
@@ -0,0 +1,254 @@
+		cli.Log.Debugw("runner region: ", "region", runner.Region)
+		cli.Log.Debugw("runner az: ", "az", runner.AvailabilityZone)
+		cli.Log.Debugw("runner instance ID: ", "instance ID", runner.InstanceID)
+		cli.Log.Debugw("runner: ", "runner hostname", runner.Runner.Hostname)
+
+		err := runner.Runner.UseIdentityFile(agentCmdState.InstallIdentityFile)
+		if err != nil {
+			return errors.Wrap(err, "unable to use provided identity file")
+		}
+
+		if err := verifyAccessToRemoteHost(&runner.Runner); err != nil {
+			cli.Log.Debugw("verifyAccessToRemoteHost failed")
+			return err
+		}
+
+		if alreadyInstalled := isAgentInstalledOnRemoteHost(&runner.Runner); alreadyInstalled != nil {
+			cli.Log.Debugw("agent already installed on host, skipping")
+			continue
+		}
+
+		token := agentCmdState.InstallAgentToken
+		if token == "" {
+			// user didn't provide an agent token
+			cli.Log.Debugw("agent token not provided")
+			var err error
+			token, err = selectAgentAccessToken()
+			if err != nil {
+				return err
+			}
+		}
+		cmd := fmt.Sprintf("sudo sh -c \"curl -sSL %s | sh -s -- %s\"", agentInstallDownloadURL, token)
+		err = runInstallCommandOnRemoteHost(&runner.Runner, cmd)
+		if err != nil {
+			cli.Log.Debugw("runInstallCommandOnRemoteHost failed")
+			return err
+		}
+	}
+
+	return nil
+}
+
+func awsCaptureTheFlagEC2IC(_ *cobra.Command, args []string) error {
+	runners, err := awsFindRunnersToCapture()
+	if err != nil {
+		return err
+	}
+
+	for _, runner := range runners {
+		cli.Log.Debugw("runner: ", "runner", runner)
+		cli.Log.Debugw("runner user: ", "user", runner.Runner.User)
+		cli.Log.Debugw("runner region: ", "region", runner.Region)
+		cli.Log.Debugw("runner az: ", "az", runner.AvailabilityZone)
+		cli.Log.Debugw("runner instance ID: ", "instance ID", runner.InstanceID)
+		cli.Log.Debugw("runner: ", "runner hostname", runner.Runner.Hostname)
+		err := runner.SendAndUseIdentityFile()
+		if err != nil {
+			cli.Log.Debugw("ec2ic failed", "err", err)
+			continue
+		}
+
+		if err := verifyAccessToRemoteHost(&runner.Runner); err != nil {
+			cli.Log.Debugw("verifyAccessToRemoteHost failed")
+			return err
+		}
+
+		if alreadyInstalled := isAgentInstalledOnRemoteHost(&runner.Runner); alreadyInstalled != nil {
+			cli.Log.Debugw("agent already installed on host, skipping")
+			continue
+		}
+
+		token := agentCmdState.InstallAgentToken
+		if token == "" {
+			// user didn't provide an agent token
+			cli.Log.Debugw("agent token not provided")
+			var err error
+			token, err = selectAgentAccessToken()
+			if err != nil {
+				return err
+			}
+		}
+		cmd := fmt.Sprintf("sudo sh -c \"curl -sSL %s | sh -s -- %s\"", agentInstallDownloadURL, token)
+		err = runInstallCommandOnRemoteHost(&runner.Runner, cmd)
+		if err != nil {
+			cli.Log.Debugw("runInstallCommandOnRemoteHost failed")
+			return err
+		}
+	}
+
+	return nil
+}
+
+func awsFindRunnersToCapture() ([]*lwrunner.AWSRunner, error) {
```

Comment:
Perhaps use AWS terminology?

```suggestion
func awsDescribeInstances() ([]*lwrunner.AWSRunner, error) {
```